cpp/include/cudf/io/json.hpp

```diff
@@ -300,6 +312,56 @@ class json_reader_options {
    */
   [[nodiscard]] json_recovery_mode_t recovery_mode() const { return _recovery_mode; }
 
+  /**
+   * @brief Whether json validation should be enforced strictly or not.
+   *
+   * @return true if it should be.
+   */
+  [[nodiscard]] bool is_strict_validation() const { return _strict_validation; }
+
+  /**
+   * @brief Whether leading zeros are allowed in numeric values. strict validation
+   * must be enabled for this to work.
+   *
+   * @note: strict_validation must be enabled for this to work.
+   *
+   * @return true if leading zeros are allowed in numeric values
+   */
+  [[nodiscard]] bool is_allowed_numeric_leading_zeros() const
+  {
+    return _allow_numeric_leading_zeros;
+  }
+
+  /**
+   * @brief Whether unquoted number values should be allowed NaN, +INF, -INF, +Infinity, Infinity,
+   * and -Infinity. strict validation must be enabled for this to work.
```

Comment:
> strict validation must be enabled for this to work.

Can you please clarify why?

Reply:
These 3 boolean options are the string validation options. If strict validation is not enabled, these 3 options are unused.